Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris_test.py

```diff
@@ -2350,3 +2350,363 @@ def test_gps_vm_recover_files_command_success(client, requests_mock, empty_respo
         assert response.raw_response == gps_vm_recover_files_response.get('raw_response')
         assert response.outputs == gps_vm_recover_files_response.get('outputs')
         assert response.readable_output == gps_vm_recover_files_hr
+
+
+def test_rubrik_sonar_user_access_list_command_success_with_empty_response(client, requests_mock):
+    """
+    Test case scenario for rubrik_sonar_user_access_list_command with valid case and empty response.
+
+    When:
+        - Calling rubrik_sonar_user_access_list_command.
+    Then:
+        - Verifies mock response with actual response.
+    """
+    from RubrikPolaris import rubrik_sonar_user_access_list_command
+
+    # Load test data
+    response_data = util_load_json(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                                "test_data/sonar_user_access_list_response.json"))
+
+    args = {"sort_order": "ASC", "limit": 1, "page_number": 1, "include_whitelisted_results": True,
+            "user_email": "demo"}
+    requests_mock.post(BASE_URL_GRAPHQL, [{"json": response_data.get('empty_response')}])
+    response = rubrik_sonar_user_access_list_command(client, args=args)
+    assert response.readable_output == MESSAGES['NO_RECORDS_FOUND'].format('user accesses')
+
+
+@pytest.mark.parametrize("limit, page_number", [(1, 1), (1, 2), (2, 1)])
+def test_rubrik_sonar_user_access_list_command_success(client, requests_mock, limit, page_number):
+    """
+    Test case scenario for rubrik_sonar_user_access_list_command with valid case.
+
+    When:
+        - Calling rubrik_sonar_user_access_list_command.
+    Then:
+        - Verifies mock response with actual response.
+    """
+    from RubrikPolaris import rubrik_sonar_user_access_list_command
+
+    # Load test data
+    response_data = util_load_json(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                                "test_data/sonar_user_access_list_response.json"))
+    if limit == 1 and page_number == 2:
+        with open(os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                               "test_data/sonar_user_access_list_response_hr_2_2.md"), 'r') as f:
+            hr_data = f.read()
+    else:
```

Comment:
Tests shouldn't contain if statements to determine which items to load or what to validate.
Let's either split this into different tests or use the parameterized to pass the file name to open.

Reply:
Updated.